### DISCUSSION TITLE
Font glyph issue: vertical colon movement

### DISCUSSION THREAD

**Discussion by notpeter** [zed-industries/zed#10527] [Feature Requests]
While coding in Rust I observed an odd behavior where the colon glyphs dance as you type them.

To reproduce type out the following:
```
fn family() -> String {
    String::from(std::consts::FAMILY)
}
```

![colon](https://github.com/zed-industries/zed/assets/145113/b88d1637-7b45-4913-b834-e4f8c4966067)

The colon(s) move twice: First, when you type the second colon (horizontally) and then when you type a letter after (vertically). No idea why the vertical movement only happens inside parens.

**Comment by notpeter** [comment]
The [Iosevka customizer](https://typeof.net/Iosevka/customizer) doesn't currently support defining variants for the ':' character, but I [opened an issue](https://github.com/be5invis/Iosevka/issues/2321) to see if they might in the future. In writing that up I also discovered the `::` sometimes dances up or dances down or just stays put depending on context. Fun!
![colon-colon2](https://github.com/zed-industries/zed/assets/145113/694bb95d-dc82-412c-9624-afa63bf465d7)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
